### ISSUE TITLE
Support for 3D images

### ISSUE THREAD

**Issue by gladomat** [Deci-AI/super-gradients#823]
### Is your feature request related to a problem? Please describe.
Medical image segmentation often relies on 3D scans (MRI, CT), but there are very few pretrained models to use for different medical image tasks (classification, segmentation)

### Describe the solution you'd like
It would be great if there were some state of the art models available in super-gradients, and even better if they would be pre-trained. For example this model http://arxiv.org/abs/2208.09567, or use the synthesized 100k brains to train a 3d classification model like this one: http://arxiv.org/abs/2209.07162, https://arxiv.org/pdf/2303.08216.pdf

**Comment by shaydeci** [comment]
Hey @gladomat, we currently prioritize other tasks and models, that are more "production oriented".
We do plan on adding models from the medical area, but a bit further down the road.
When the time comes will consider your feature request (:
